src/utils/eventTable.tsx

```diff
@@ -74,24 +95,76 @@ const getFamilyEvents = (family: TFamilyPublic): TEventWithDocument[] =>
 
 export const getEventTableRows = ({
   families,
+  familyTopics,
   documentEventsOnly = false,
   matchesFamily,
   matchesStatus = "success",
   language,
 }: {
   families: TFamilyPublic[];
+  familyTopics?: IFamilyDocumentTopics;
   documentEventsOnly?: boolean;
   matchesFamily?: TMatchedFamily;
   matchesStatus?: TLoadingStatus;
   language?: string;
 }): TEventTableRow[] => {
   const rows: TEventTableRow[] = [];
+  const topicsData = familyTopics ? Object.values(familyTopics.conceptsGrouped).flat() : [];
 
   families.forEach((family) =>
     getFamilyEvents(family).forEach(({ event, document }, eventIndex) => {
       if (documentEventsOnly && !document) return;
 
       const date = new Date(event.date);
+      const summary = event.metadata.description?.[0];
+
+      const linkClasses = "block text-gray-700 underline underline-offset-4 decoration-gray-300 hover:decoration-gray-500";
+
+      /* Topics */
+
+      let topicsDisplay: ReactNode = null;
+      if (document && familyTopics) {
+        const documentTopicsData = familyTopics.documents.find((doc) => doc.importId === document.import_id)?.conceptCounts ?? {};
+
+        const sortedTopics = orderBy(Object.entries(documentTopicsData), ["1"], ["desc"]);
+        const someTopicsHidden = sortedTopics.length > MAX_TOPICS_PER_DOCUMENT;
+
+        const topicLinks = sortedTopics.slice(0, MAX_TOPICS_PER_DOCUMENT).map(([topicId, topicCount]) => {
+          const [wikibaseId, fallbackLabel] = topicId.split(":");
```

Comment:
I've had to do similar in my branch... Not sure if it is worth splitting them out at the process level or not. Let's leave it for now and circle back